source/site_ops/how-tos/enable_notification_tray.rst

```diff
@@ -0,0 +1,134 @@
+.. _Enable Notification Tray:
+
+Enable Notification Tray
+########################
+
+.. tags:: site operator, how-to
+
+The **notification tray** allows learners to access platform notifications from
+the top-right corner of the Open edX interface.
+
+The recommended way to enable notifications is by using the official Tutor
+plugin:
+`tutor-contrib-platform-notifications <https://github.com/openedx/tutor-contrib-notifications>`_.
+
+Install and enable the plugin
+*****************************
+
+Install the plugin:
+
+.. code-block:: bash
+
+   pip install tutor-contrib-platform-notifications
+
+Enable the plugin in your Tutor environment:
+
+.. code-block:: bash
+
+   tutor plugins enable notifications
+
+When enabled, the plugin automatically:
+
+- Enables the following waffle flags:
+
+  - ``notifications.enable_notifications``
+  - ``notifications.enable_email_notifications``
+
+- Sets the required environment variables for the notifications service
+
+.. important::
+
+   After enabling the notifications plugin, site operators **must rebuild the
+   MFE image** for the notification tray to appear in the user interface.
+
+Rebuild the MFE image:
+
+.. code-block:: bash
+
+   tutor images build mfe
+
+
+Configure the notifications plugin
+**********************************
+
+The notifications plugin exposes several configuration options that can be
+customized using Tutor.
+
+You can set these values either:
+
+- In ``config.yaml``, or
+- Using the ``tutor config set`` command
+
+For more information, see
+`Tutor configuration <https://docs.tutor.edly.io/configuration.html#configuration>`_.
+
+
+NOTIFICATIONS_DEFAULT_FROM_EMAIL
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Please use `=` for third level (we do headers by descending order with number of lines: # (4 lines), * (3 lines), = (two lines), - (1 line))